### PR TITLE
fix #9, gray out done task

### LINKED ISSUE #9: "gray out" a completed task
Currently, when the user marks a task as "complete", the only changes on the UI is that the content text of the task gets a line through. A more impactful change needs to be done, changing the color of the panel to a more darker one seems appropiate.

### PR DESCRIPTION
**Pull request by vuchl**
Hey @DevTony101 !

I saw the issue posted and created this fix.

Regards Sebastian

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ document.addEventListener("DOMContentLoaded", () => {
 
         // Message container
         const message = document.createElement("article");
-        message.classList.add("message", "is-primary");
+        message.classList.add("message", done ? "is-secondary" : "is-primary");
```
Comment (DevTony101) [inline src/index.js:59]: There is no "is-secondary" color class in bulma. Perhaps you meant to say "is-dark"?

File: src/index.js [hunk 2]
```diff
@@ -147,6 +147,8 @@ document.addEventListener("DOMContentLoaded", () => {
     const id = Number(task.dataset.id);
     database.toggleDone(id, isDone, () => {
       task.classList.toggle("is-done", isDone);
+      task.classList.toggle("is-primary", !isDone);
+      task.classList.toggle("is-secondary", isDone);
```
Comment (DevTony101) [inline src/index.js:151]: There is no "is-secondary" color class in bulma. Perhaps you meant to say "is-dark"?

**Review by DevTony101 — CHANGES_REQUESTED** [review]
Greetings sebastian! Thank you so much for your contribution! Your code seems to work but there are some things you might want to correct, please refer to the comments I made.

File: src/index.js (hunk 2 above)
Comment (vuchl) [reply src/index.js]: Fixed in #36699a6

File: src/index.js (hunk 1 above)
Comment (vuchl) [reply src/index.js]: Fixed in #36699a6. Sorry for the mistake. it looke dark in the dev server 😃

**Review by DevTony101 — APPROVED** [review]
Good job! Thanks again for contributing 👍🏽

**Pull request merged**